packages/vitest/src/utils/coverage.ts

```diff
@@ -272,6 +273,37 @@ export class BaseCoverageProvider {
       return chunks
     }, [])
   }
+
+  createUncoveredFileTransformer(ctx: Vitest) {
+    const servers = [
+      ...ctx.projects.map(project => ({
+        root: project.config.root,
+        vitenode: project.vitenode,
+      })),
+      // Check core last as it will match all files anyway
+      { root: ctx.config.root, vitenode: ctx.vitenode },
+    ]
+
+    return async function transformFile(filename: string) {
+      let lastError
+
+      for (const { root, vitenode } of servers) {
+        if (!filename.startsWith(root)) {
+          continue
+        }
+
+        try {
+          return await vitenode.transformRequest(filename)
```

Comment:
What if the workspace uses browser?

Reply:
I think it works the same with vite node here. For uncovered files we don't need to worry about the wrapper length as these are not executed.